nearcore/src/config.rs

```diff
@@ -468,7 +476,14 @@ impl Config {
             .unwrap_or_else(|_| panic!("Could not open config file: `{}`", path.display()));
         let mut content = String::new();
         file.read_to_string(&mut content).expect("Could not read from config file.");
-        Config::from(content.as_str())
+        let config = Config::from(content.as_str());
+        if config.num_epochs_to_keep_store_data < MIN_NUM_EPOCHS_TO_KEEP_STORE_DATA {
+            panic!(
+                "num_epochs_to_keep_store_data: {} is below: {}",
+                config.num_epochs_to_keep_store_data, MIN_NUM_EPOCHS_TO_KEEP_STORE_DATA
+            );
+        }
```

Comment:
Is this the right place to put this validation? Seems rather easy to dodge by, eg, parsing from string directly.

Reply:
My gut feeling is that it makes sense to spend effort to make such validation tight. In this case, I'd say that the most robust and simplest thing would be to just add 

```rust
let num_epochs_to_keep_store_data = num_epochs_to_keep_store_data.max(MIN_NUM_EPOCHS_TO_KEEP_STORE_DATA)
```

to the NightshadeRuntime::new